doozer/doozerlib/golang_builder_FIPS_wrapper.sh

```
@@ -314,6 +323,23 @@ if [[ "${EXEMPT}" != "1" ]]; then
       fi
     fi
     ARGS+=("${arg}")
+
+    # When coverage is enabled and explicit .go files are being compiled,
+    # ensure coverage_server.go is included if it exists alongside them.
+    # Package-path builds (e.g. ./cmd/...) don't need this since Go
+    # automatically includes all .go files in the package.
+    if [[ "${GO_COMPLIANCE_COVER}" == "1" && "${IN_BUILD}" == "1" && "${COVERAGE_ADDED}" == "0" && "${arg}" == *.go ]]; then
```

Shellcheck (0.11.0)

[warning] In POSIX sh, [[ ]] is undefined.

(SC3010)